bin/md2gslides.js

```diff
@@ -28,7 +28,11 @@ const opener = require('opener');
 const readline = require('readline');
 
 const SCOPES = [
-    'https://www.googleapis.com/auth/presentations'
+    'https://www.googleapis.com/auth/presentations',
+    'https://www.googleapis.com/auth/drive',
```

Comment:
Doesn't make sense to have the drive scope *and* all the other scopes. The Drive scope is inclusive of all of them.

Just the drive scope is sufficient here. It's a bit too broad, but it's the only one that would allow copying an arbitrary file.